agent/src/main/scala/za/co/absa/atum/agent/dispatcher/HttpDispatcher.scala

```diff
@@ -30,13 +30,14 @@ class HttpDispatcher(config: Config) extends Dispatcher with Logging {
   logInfo("using http dispatcher")
   logInfo(s"serverUri $serverUri")
 
-  override def getOrCreateAtumContext(partitioning: PartitioningDTO): AtumContextDTO = {
+  override def getOrCreateAtumContext(partitioning: PartitioningSubmitDTO): AtumContextDTO = {
     basicRequest
       .body(s"$partitioning")
       .post(serverUri)
       .send(backend)
 
-    AtumContextDTO(partitioning = partitioning.partitioning) // todo: implement request
+    // todo: implement request
```

Comment:
Fotgotten TODO? Or planned in following PR?

Reply:
Not forgotten, @benedeki  is gonna work on it soon: https://github.com/AbsaOSS/atum-service/issues/52